node_2026/upload_s3.py

```
@@ -0,0 +1,145 @@
+            resource.meta.client.upload_file(uploadfile, BUCKET, uploadkey)
+        except Exception as e:
+            log.critical('error uploading to S3: ' + str(e))
+            return
+
+        try:
+            os.remove(os.path.join(path, filename))
+        except FileNotFoundError:
+            pass  # ffmpeg may have already replaced or rotated the file
+        except Exception as e:
+            log.warning('error removing local file: ' + str(e))
+    except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)